Popup/explainer.md

```diff
@@ -49,40 +49,69 @@ We propose a new HTML element called `popup`. This new element can be used for a
 
 `popup` will include:
 
-* A `show()` JavaScript method, for invoking the `popup`. When visible, the `popup` will have a default set of behaviors as well as a default positioning scheme.
+* A few options to invoke/show the `popup`:
+  * A `popup` attribute, applied to whichever element should invoke a given popup (if applicable).
+  * An optional `option` attribute, applied to the `popup` to express that it should be shown.

```

Comment:
```suggestion
  * An optional `open` attribute, applied to the `popup` to express that it should be shown.
```

Reply:
Haha wow, my brain completely read the original as `open`. This is why we have reviewers! :)